Updated scarab to include info button

mkong0216 opened the pull request:
Added info button to scarab and pop up description box when info button is clicked in response to issue #1

Commit:
Update scarab.js

louh commented on src/scarab.js:
```diff
@@ -152,6 +171,32 @@ var MapzenScarab = (function () {
+  }
+
+  function _buildDescription() {
+    var elem = document.getElementById('description')
```
Great start so far, @mkong0216! I'll have some other suggestions later, but the part we need to take care of here is this `#description` element. This script assumes that it exists on the page, but it might not. We'll have to construct this element and append it to the page, like the rest of scarab.

Commit:
Update scarab.js

Modified code so _buildDescription() creates #description element and then adds the given description text

Commit:
Update scarab.css

Modified info box and button styling

Commit:
Update scarab.js

Changed description box's class name to 'mz-bug-description'

Commit:
Update scarab.css

changed description to 'mz-bug-description

louh commented on src/scarab.css:
```diff
@@ -150,6 +179,39 @@
+}
+
+.mz-bug-info-logo {
+  background-image: url(http://png-1.vector.me/files/images/3/9/394040/universal_information_symbol_thumb);
```
I think we need to do a little bit of extra homework on this image. We need to make sure we have the rights to use and distribute it -- or we might create our own. Ask @souperneon for help on this.

Once we have the correct asset, we need to host it in this repository. (3rd party websites can go down or move files without our knowledge, so we can't depend on using those resources.) We can either put the file in this directory (https://github.com/mapzen/scarab/tree/master/src/images) where the Mapzen logo already is, or embed it as a data-URL like other images here.

souperneon replied:
I believe @mkong0216 is using FontAwesome for now

Commit:
Changed calculation of container width

Created new function to check current width of 'mz-bug-container' and then adds on the info button width to the container if needed (if internal stylesheets are used to change container width, it will take into account the new widths). Also used font-awesome to include info button image

Commit:
Using font-awesome

Uses font-awesome to include the info button image

Commit:
No longer need left styling

Changed way scarab displays its icons to float: left 
No longer need to calculate how far left each icon needs to be
